pb/summary/summary.proto

```diff
@@ -77,6 +78,9 @@ message TestInfo {
   // The flakiness of the test, measured out of 100
   float flakiness = 8;
 
+  // The flakiness of the test from the previous interval
+  float previous_flakiness = 10;
```

Comment:
Consider having this be repeated, if we'd ever be interested in looking at or comparing multiple previous flakiness values? (Can have it limited in the code so it doesn't grow unbounded, e.g. for now we can make it repeated but in the code ensure we only assign 1 value to it ever).

Reply:
Okay, done